src/pages/Embark/index.tsx

```diff
@@ -196,6 +197,11 @@ export const EmbarkRoot: React.FunctionComponent<EmbarkRootProps> = (props) => {
     <StorageContainer>
       {(storageState) => (
         <EmbarkProvider
+          externalRedirects={{
+            Offer: () => {
+              history.push('/new-member/offer')
```

Comment:
use a constant for this maybe, so that we don't have to change it in two places :)

Reply:
just checking does this work with `/en/new-member` also?

Reply:
I don't know 😞 

I'll make sure to try it, there are some things I need to do before I can check this one off.

I'll mark it as do not merge!